docker-compose.yml

```diff
@@ -40,50 +40,19 @@ services:
     api:
       container_name: collector_api
       build: .
-      image: cyfacede/collector:2.0.0
-      ports:
-        - "127.0.0.1:8080:8080" # 127.0.0.1 avoids the manipulation of the host's ip-tables
-        - "127.0.0.1:13371:13371"
+      image: cyfacede/collector:2.0.2
+      ports: # It is ok that this entry manipulates the host's ip-tables since this compose file should only be used in dev environments.
```

Comment:
We should mention this in the Readme then and point to the deployment docker-compose file (if this is publicly available), or else mention that the deployment d-c file is not publicly available.